index.html

```diff
@@ -69,7 +69,6 @@
       caniuse: "nav-timing",
       mdn: "navigation-timing",
       xref: {
-        url: "https://respec.org/xref",
```

Comment:
This seemed to break the build. Not sure why, and can't find any associated change in https://github.com/speced/respec/ but this line isn't in Resource Timing and doesn't seem to be needed.